test/ruleLoaderTests.ts

```diff
@@ -18,6 +18,7 @@ import * as Lint from "./lint";
 describe("Rule Loader", () => {
     const path = require("path");
     const rulesDirectory = path.join(global.process.cwd(), "build/rules");
+    const rulesDirectories = [rulesDirectory];
```

Comment:
sorry, I meant use the value `[rulesDirectory]` inline where the variable is referenced, rather than creating this extra value. but that's a small nitpick; I won't let it block the PR